core/server/api/shared/http.js

```diff
@@ -21,7 +21,8 @@ const http = (apiImpl) => {
 
         frame.configure({
             options: apiImpl.options,
-            data: apiImpl.data
+            data: apiImpl.data,
+            rawInclude: apiImpl.rawInclude
```

Comment:
have you thought about a skip strategy for shared serializers?

Reply:
Do you mean conditionally skipping this step in serialization process - https://github.com/TryGhost/Ghost/blob/master/core/server/api/shared/serializers/handle.js#L28 ?

Not sure I fully understand how that would be done within the frame :thinking: ? Can you please expand?

Reply:
moved this logic to `handle.js` and now conditionally skipping input serialization logic based on `frame.options.rawInput` flag -> 987eb057a99181c05dec17f2f43d0c7703070e9e